index.bs

```diff
@@ -990,6 +990,12 @@ However, it will still be possible for some, or all, hints to be requested and u
 [=user agents=] should consider policies to restrict or reduce access to parties that are known to
 <a lt="active fingerprinting">actively fingerprint</a> their users.
 
+[=User agents=] should take care to not introduce fingerprinting vectors through GREASE-like brand
+lists that might be unique for an individual or a very small group of users. Rather, a strategy
+should be employed where the arbitrary brand is either shared across many users (e.g., stable across
+major versions for all users) or unstable and unpredictable (e.g., regenerated per-request). However
```

Comment:
Do we need the "unstable and unpredictable" variant, given that it'd be caching hostile?

Reply:
I think I'll remove that. In theory a UA could take a stance where randomization is more important than cache-hits, but maybe we shouldn't encourage it (especially since per-release randomization is good enough IMHO).